samples/wang_landau_reaction_ensemble.py

```diff
@@ -17,7 +17,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 """
-This sample simulates the Wang-Landau Reaction Ensemble for a harmonic bond.
+This example script simulates two reacting monomers which are bonded via a harmonic potential.
+The script aborts as soon as the abortion criterion in the Wang-Landau algorithm is met.
+The Wang-Landau potential which is updated during the simulation is written to the file WL_potential_out.dat.
```

Comment:
Just to check my understanding: we have a neutral monomer (type 3) and a base monomer (type A-) connected by a bond, and two protons (type H+) to set the pH. The WLRE algorithm changes the base monomer into its conjugated acid (type AH) and back using MC moves and stores the degree of association ñ and the chemical potential µ to two files.

Reply:
The situtation is the following: We have two bonded monomers. These can change their type. They are either of type A- or of type HA. When they are of type A- they have released the corresponding H+ ion. The pH would have to be determined through the reweighted H+ concentration after the simulation is over.
The Wang-Landau algorithm is a histogram reweighting algorithm and drives the system out of equilibrium via applying a Wang-Landau potential. This is not directly connected to the chemical potential. The Wang-Landau potential either represents the canonical partition function, the density of states or the a mixture of both, depending on the acceptance probabilities you impose on the system.
A detailed explaination can be found in https://pubs.acs.org/doi/full/10.1021/acs.jctc.6b00791 .
With the simulation setup configured in this script the

Reply:
Thanks!